src/content/docs/guides/JSON/2-json_writing.mdx

```diff
@@ -95,6 +126,8 @@ json_set_object(new_game_data, "screenSize", screen_size_data);
 </TabItem>
 
 <TabItem label="C#">
+<Tabs syncKey="csharp-style">
+<TabItem label="Top-level Statements">
 
 ```csharp
 Json screenSizeData = CreateJson();
```

Comment:
I think showing the use of `using` static `SplashKitSDK.SplashKit;` in top level code could be a good idea which will make it easier to understand. Its just a suggestion.

Reply:
We discussed this statement earlier, and for the sake of compatibility with other tutorials in this series, we do not use static statements